Sources/SpeziBluetoothServices/Characteristics/PLXFeatures.swift

```diff
@@ -0,0 +1,101 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ByteCoding
+import Foundation
+import NIOCore
+import SpeziNumerics
+
+
+public struct PLXFeatures: ByteCodable, Hashable, Sendable, Codable {
```

Comment:
Maybe we can at least add some type-level documentation so it shows up nicely in DocC.